src/diffusers/schedulers/deprecated/scheduling_dpmsolver_multistep_legacy.py

```diff
@@ -0,0 +1,842 @@
+# Copyright 2023 TSAIL Team and The HuggingFace Team. All rights reserved.
```

Comment:
I don't think it makes sense to create a whole new "legacy" scheduler class here. I'd advocate for just deprecating it in the original file

Reply:
ok! will do that instead. I was just kinda eager to get rid of the code sooner 😛